src/local/butler/scripts/backtracker.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""backtracker.py backtracks GitHub issues filed in the past,
+        update their information to cloud storage if missing
+        and close issues when necessary."""
+
+from clusterfuzz._internal.datastore import data_types
+from libs.issue_management import oss_fuzz_github
+
+
+def _get_testcase(bug_information):
+  """Retrieve testcase based on its bug information."""
+  candidates = list(
+      data_types.Testcase.query(
+          data_types.Testcase.bug_information == bug_information))
+  if not candidates:
+    print('No candidate found when '
+          'querying bug information {bug_information}\n')
+  elif len(candidates) > 1:
+    print('Multiple candidates found when '
+          f'querying bug information {bug_information}:\n')
+    for testcase in candidates:
+      print(f'  {testcase.key.id()}')
+  else:
+    return candidates[0]
+  return None
+
+
+def _testcase_information_verified(testcase, issue):
+  """Verify if a testcase correctly stores its GitHub issue information."""
+  if testcase.github_repo_id == issue.repository.id and \
+          testcase.github_issue_num == issue.number:
+    print(f'Testcase {testcase.bug_information} was properly stored.')
+    return True
+
+  if testcase.github_repo_id is None and testcase.github_issue_num is None:
+    print(f'Testcase {testcase.bug_information} was not stored.')
+  else:
+    print(f'Testcase {testcase.bug_information} stored '
+          f'is inconsistent with GitHub:\n'
+          f'  Issue number  (Storage) {testcase.github_issue_num} '
+          f'!= {issue.number} (GitHub)\n'
+          f'  Repository ID (Storage) {testcase.github_repo_id} '
+          f'!= {issue.repository.id} (GitHub).')
+  return False
+
+
+def execute(args):
+  """Backtrack GitHub issues filed in the past,
+    update their information in gcloud, and close them when necessary."""
+  for issue in oss_fuzz_github.get_my_issues():
+    # Track testcase.
+    bug_information = oss_fuzz_github.get_bug_information(issue)
+    if bug_information is None:
```

Comment:
nit: do `if not bug_information`. 

See https://google.github.io/styleguide/pyguide.html#214-truefalse-evaluations